src/webgl/loading.js

```diff
@@ -9,6 +9,183 @@
 import p5 from '../core/main';
 import './p5.Geometry';
 
+/**
+ * Load a 3d model from an OBJ or STL string.
+ *
+ * <a href="#/p5/loadModel">createModel()</a> should be placed inside of <a href="#/p5/preload">preload()</a>.
```

Comment:
We can probably take this sentence out since I think we don't do any preloading in this method?